assets/components/LogViewer/SimpleLogItem.vue

```diff
@@ -9,14 +9,16 @@
     ></div>
     <log-message-actions
       class="duration-250 absolute -right-1 opacity-0 transition-opacity delay-150 group-hover/entry:opacity-100"
-      :message="logEntry.message"
+      :message="decodeXML(stripAnsi(logEntry.message))"
```

Comment:
Hi @amir20 ! I did this here and not in the `LogMessageActions` component itself, since the component handles `JSON` string also.

Reply:
I was just thinking of something. This might be a performance issue since it is being computed even user doesn't need to copy. 

Although, probably tiny and just not a big deal. Something to measure before releasing.

Reply:
> I was just thinking of something. This might be a performance issue since it is being computed even user doesn't need to copy.

@amir20 , maybe we can implement this in the `LogMessageActions` widget itself?

Reply:
I think it's a micro optimization. But if we wanted to it can be passed as a lazy function. Then with toValue converted to real value. 

We should measure first before doing too much work.